core/controllers/base_test.py

```diff
@@ -107,6 +108,8 @@ class BaseHandlerTests(test_utils.GenericTestBase):
 
     class MockHandlerWithInvalidReturnType(base.BaseHandler):
         GET_HANDLER_ERROR_RETURN_TYPE = 'invalid_type'
+        URL_PATH_ARGS_SCHEMAS = {}
+        HANDLER_ARGS_SCHEMAS = {'GET': {}}
```

Comment:
**Note to Reviewers:** Since this PR is adding the architecture for validating the handler params, schema is required for MockHandlers, else Schema-Validation-System (SVS) architecture will raise NotImplementedError for them.